docs/pages/enterprise/workflow/ssh-approval-slack.mdx

```diff
@@ -76,15 +80,28 @@ $ tctl create -f access.yaml
 
 #### Export access-plugin Certificate
 
-Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://gravitational.com/teleport/docs/cli-docs/#tctl-auth-sign).
-
-```bash
-$ tctl auth sign --format=tls --user=access-plugin --out=auth --ttl=720h
-# ...
-```
-
-The above sequence should result in three PEM encoded files being generated: auth.crt, auth.key, and auth.cas (certificate, private key, and CA certs respectively).  We'll reference the auth.crt, auth.key, and auth.cas files later when [configuring the plugins](#configuring-teleport-slack).
+Teleport Plugin use the `access-plugin` role and user to perform the approval. We export the identity files, using [`tctl auth sign`](https://goteleport.com/docs/cli-docs/#tctl-auth-sign).
 
+<Tabs>
+<TabItem label="Teleport Cloud or via Proxy connection">
+  If you are using Teleport Cloud or want to have the plugin only connect through the proxy ports you must use a identity file.  When you `tctl auth sign` through Teleport Cloud or remotely you must have [impersonation](https://goteleport.com/docs/access-controls/guides/impersonation/) rights on your user's role.
+
+  ```bash  
+  $ tctl auth sign --user=access-plugin --format=file --out=access-plugin.pem --ttl=720h
+  # ...
+  ```  
+  The above sequence will result in one PEM encoded files being generated: access-plugin.pem (certificate, private key, and CA certs respectively).  We'll reference the access-plugin.pem later when [configuring the plugins](#configuring-teleport-slack).
+</TabItem>
+<TabItem label="On Premise">
```

Comment:
Nit: we don't have On Premise as a term. Let's use Open Source and Enteprise even at the cost of duplication, let's add two tabs.